tungsten_ci_utils/test_statistics/requirements.txt

```diff
@@ -0,0 +1,5 @@
+configparser
```

Comment:
I think that we should pin versions in our requirements even if they are old. That could save us a lot of time in the future. :)